data/events.json

```diff
@@ -1,581 +1,1302 @@
+		"location": "Main Stage",
+		"timestamp": 1,
+		"imageUrl": "",
+		"facebookEventLink": "",
+		"additionalInfo": [],
+		"categories": [
+		    "main stage and pro shows"
+		],
+		"organizers": [
+		    {
+			"name": "",
+			"phoneNumber": 0
+		    },
+		    {
+			"name": "",
+			"phoneNumber": 0
+		    }
+		]
+	},
+
+	{
+		"id": 151,
+		"name": "EDM Night",
+		"description": "The rage of today-128 BPM big room rage stuff from the likes of Tieso, Hardwell, Martin Garrix and so on. Performed under SunBurn by renowned artist such as AceAxe, SickLead and more in the past, guess what lies ahead!",
+		"location": "Main Stage",
+		"timestamp": 1,
+		"imageUrl": "",
+		"facebookEventLink": "",
+		"additionalInfo": [],
+		"categories": [
+		    "main stage and pro shows"
+		],
+		"organizers": [
+		    {
+			"name": "",
+			"phoneNumber": 0
+		    },
+		    {
+			"name": "",
+			"phoneNumber": 0
+		    }
+		]
+	},
+
+	{
+		"id": 152,
+		"name": "Humour Night",
+		"description": "Make your days less gloomy and your life a little happier. Come join us this autumn where we introduce the Humour Night.",
+		"location": "Main Audi",
+		"timestamp": 1,
+		"imageUrl": "",
+		"facebookEventLink": "",
+		"additionalInfo": [],
+		"categories": [
+		    "main stage and pro shows"
+		],
+		"organizers": [
+		    {
+			"name": "",
+			"phoneNumber": 0
+		    },
+		    {
+			"name": "",
+			"phoneNumber": 0
+		    }
+		]
+        },
+
+	{
+		"id": 153,
+		"name": "Bollywood Night",
+		"description": "Bollywood is a religion and the devotees are plenty. Get ready for whats in store this year as the legacy from Daler Mehandi, Sona Mahapatra, Neha Kakkar to Benny Dayal follows.",
+		"location": "Main Stage",
+		"timestamp": 1,
+		"imageUrl": "",
+		"facebookEventLink": "",
+		"additionalInfo": [],
+		"categories": [
+		    "main stage and pro shows"
+		],
+		"organizers": [
+		    {
+			"name": "",
+			"phoneNumber": 0
+		    },
+		    {
+			"name": "",
+			"phoneNumber": 0
+		    }
+		]
+	},
+
+	{
+		"id": 154,
+		"name": "Headliner",
+		"description": "The standard has been set high with The Local Train performing in the previous edition and bigger and better can surely be expected in this edition.",
+		"location": "Main Stage",
+		"timestamp": 1,
+		"imageUrl": "",
+		"facebookEventLink": "",
+		"additionalInfo": [],
+		"categories": [
+		    "main stage and pro shows"
+		],
+		"organizers": [
+		    {
+			"name": "",
+			"phoneNumber": 0
+		    },
+		    {
+			"name": "",
+			"phoneNumber": 0
+		    }
+		]
+	},
+
+	{
+		"id": 155,
+		"name": "A'La Mode",
+		"description": "High on the Fashion quotient and too Glam to give a Damn, the Fashionistas eccentricate the stage in all colours and outfits in the Fashion Show at Effervescence'17.",
+		"location": "Main Stage",
+		"timestamp": 1,
+		"imageUrl": "",
+		"facebookEventLink": "",
+		"additionalInfo": [],
+		"categories": [
+		    "main stage and pro shows"
```

Comment:
Make it as 2 different strings in the array rather than an 'and'. Do check in all other places if similar. Rest looks good.